tensorboard/plugins/graph/tf_graph_app/tf-graph-app.ts

```diff
@@ -199,6 +201,11 @@ class TfGraphApp extends LegacyElementMixin(PolymerElement) {
     }
   }
   _fit() {
-    (this.$$('#graphboard') as any).fit();
+    ((this.$$('#graphboard') as unknown) as TfGraphBoard).fit();
```

Comment:
Why do we need to cast through `unknown`s here? It works fine for me
locally if casting directly to `TfGraphBoard`.

Reply:
Ah true, it was actually unnecessary.  Even with `as TfGraphBoard`, the internal sync seems to not like this, so I've set it back to `as any` for now.

On that note, I still haven't figured out how to trigger the internal check in OSS, as simply adding compiler option `declaration: true` doesn't do the trick.